web/elm/src/TopBar/TopBar.elm

```diff
@@ -601,10 +599,9 @@ viewDropdownItems { query, dropdown } model =
                             ]
 
                         "team:" ->
-                            model.teams
-                                |> RemoteData.withDefault []
+                            groups
                                 |> List.take 10
-                                |> List.map (\t -> "team: " ++ t.name)
+                                |> List.map (\g -> "team: " ++ g.teamName)
```

Comment:
Are groups and teams actually equivalent?

Reply:
yeah! groups are actually better, because they're the source of truth for what's on the dashboard, which is conceptually what you're actually filtering. It's this change that fixes #3353.

Reply:
Good to hear!